### DISCUSSION TITLE
<title>Getting a 404 error in console for images

### DISCUSSION THREAD

**Discussion by lfgraphics** [community/community#41864] [Projects and Issues]
### 404 Error

General

### Body

I'm getting a 404 error error in console and images or favicon of the page isn't being loaded
[page url](https://iftikharulmakatib.github.io/dailycontest/) - [Repo url](https://github.com/iftikharulmakatib/dailycontest) Please help me out

![Screenshot (19)](https://user-images.githubusercontent.com/96362790/207954519-74e717f4-7467-482a-9aa9-66aa26d3df5a.png)
![Screenshot (20)](https://user-images.githubusercontent.com/96362790/207954604-ebeaf3c4-7ff1-4e4e-b5c0-f61d9100fd70.png)

**Comment by radar-blip** [comment]
If you take the first slash out of this link `<img class="logo" src="/images/images/logo.png"> ` it might help.
I'm not for sure but its worth a try.

**Reply by lfgraphics** [reply]
thanks for your contribution but it didn't worked and not only images, the same problem is for every static file like png, menfist and ttf files also

**Reply by DSDmark** [reply]
Did you fix your issue? There is no error at all.

![image](https://user-images.githubusercontent.com/85744816/208113016-0d51f7ec-4a7c-41eb-9d82-12a6c5596300.png)

**Comment by getlost01** [comment]
do `<img class="logo" src="./media/images/logo.png" alt=".">`    it works perfectly.
